lib/constants.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+#  Constants for this library.
+module EasyPost::Constants
+  CA_TYPES_WITH_CUSTOM_FLOWS = %w[FedexAccount UpsAccount].freeze
```

Comment:
Please be explicit, no acronyms.

Reply:
Ruby complains about too long of a variable name